.github/workflows/ci_cron.yml

```diff
@@ -318,7 +318,7 @@ jobs:
     name: Notify slack success
     needs: [unit_tests, multiverse, infinite_tracing]
     runs-on: ubuntu-22.04
-    if: always()
+    if: always() && ${{ github.event_name != 'workflow_dispatch' }}
```

Comment:
I couldddd see us wanting to know when a re-run turns green.

Reply:
Hmmm I don't think it would do that if i'm understanding it right. When we hit the rerun button on a failed job, i think it should still use the original event/event name. it seems like thats what its describing [here](https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs) although it doesn't explicitly mention the event_name.

As far as i can tell, this would only change the behavior for when we run it but hittin the "run workflow" button on the actions page (like [here](https://github.com/newrelic/newrelic-ruby-agent/actions/workflows/ci_cron.yml)). 

So I think rerunning a failed nightly run should still notify slack, I could be wrong though.

Reply:
(discussed in slack)